content/docs/started/getting-started-gke.md

```diff
@@ -72,8 +72,15 @@ Create an OAuth client ID to be used to identify Cloud IAP when requesting acces
     export CLIENT_ID=<CLIENT_ID from OAuth page>
     export CLIENT_SECRET=<CLIENT_SECRET from OAuth page>
     ```
+## Deploy Kubeflow on Kubernetes Engine Using The Ui
 
-## Deploy Kubeflow on Kubernetes Engine
+1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy) in your web browser
```

Comment:
For consistency with the rest of the page, we should add full stops at the end of each sentence, and bold highlighting to the UI controls. Something like this (I've also added a line in response to @kunmingg 's comment):

```
1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy) in your web browser.
1. Sign in using the account that owns your GCP project.
1. Complete the form.
1. Click **Create Deployment**.
```